_sass/_uswds-custom-styles.scss

```diff
@@ -0,0 +1,164 @@
+// ==========================================================================
+// Custom style overrides
+// This file should be imported after your USWDS theme settings overrides.
+// If you don't have a setting available to change, you'd override here.
+// Meant for manually overriding component styles found in:
+// https://github.com/uswds/uswds/tree/develop/src/stylesheets/components
+// ==========================================================================
+
+// ==========================================================================
+// Header
+// ==========================================================================
+.usa-header {
+
+  .usa-logo__text a {
+    @include add-background-svg("DHS_4R_Blue", "../../img");
+    background-size: cover;
+    display: inline-block;
+    // px-to-rem() is a built-in USWDS function.
+    height: px-to-rem(31px);
+    width: px-to-rem(106px);
+
+    @include at-media($theme-header-min-width) {
+      height: px-to-rem(72px);
+      width: px-to-rem(247px);
+    }
+  }
+}
```

Comment:
5. Now that we've removed the inline image we need to set logo as a background image, with it's respective sizing on small and large screens.

```scss
@include add-background-svg("DHS_4R_Blue", "../../img");
```

A built-in USWDS mixin that let's us add a SVG as well as define a custom path (otherwise it defaults to `$theme-image-path: "../img";` in `_uswds-theme-general.scss`.

We're also using a function that allows us to easily use recognizable PX values, but converts them to more modern REM unit values.

```scss
px-to-rem(31px)
```